### ISSUE TITLE
etc/hostname: Permission denied

### ISSUE THREAD

**Issue by kegra** [dale3h/hassio-installer#9]
used sudo su , i can use nano to change anything so i dont see why this is not running.
root@hassio:~# curl -sL https://raw.githubusercontent.com/dale3h/hassio-installer/master/hassio_rpi3bp | bash -s
main: line 40: /etc/hostname: Permission denied
sed: -e expression #1, char 8: unknown option to `s'

**Comment by simonfell** [comment]
Hi
I am getting this issue, but with being new to all this and trying to learn,  i am struggling to see where i can change line 40.  I can SSH to my Pi3b+.  has an edited script been done.  Thanks for any help

**Comment by marduv** [comment]
@simonfell I have posted a fix, but are waiting for @dale3h to approve.
Meanwhile you can use:
`curl -sL https://raw.githubusercontent.com/marduv/hassio-installer/master/hassio_rpi3bp | bash -s`

**Comment by Jafterdark** [comment]
I tried using this one but then I lose the Hassio tab and addons and I did go in the configuration.yaml file and add it but that doesn't work either
curl -sL https://raw.githubusercontent.com/marduv/hassio-installer/master/hassio_rpi3bp | bash -s

**Comment by marduv** [comment]
@Jafterdark Is this an existing install or did you make it from scratch?
In `configuration.yaml` it is important that you only have `hassio:` mention once.

This script dos not do any thing else then, add depednencys, install `Docker` and run `hassio_install` script.
But if you have a instance already installed before trying this, then the probability is preity hi that you are running two instances.

**Comment by Jafterdark** [comment]
In configuration.yaml it is important that you only have hassio: mention once. I have it only mentioned once it could be just due to the new version but at this time I cannot get to it at all even if I do /dashboard

**Comment by frankhommers** [comment]
@marduv, it doesn't seem to work for me. I don't know whats wrong. It all seems to work, and in the end I get the message to access it by http://10.0.0.149:8123. 

But that is unavailable, even after reboot. What could possibly be wrong?

**Comment by marduv** [comment]
@Jafterdark Sorry for late replay :disappointed:
If you can see the Hass.IO tab, it means the addon is not running an you cant access it just by typing inn the url.

Your issue is not a caused by this script, it use [Hass.IO Builder](https://github.com/home-assistant/hassio-build) to install Hass.IO.
This script does nothing more than install necessary dependencies and run Hass.IO builder setup.

So if you have a clean setup, and there is nothing else wrong with your config file then you need to take a look at [Hass.IO GIT Reposotory](https://github.com/home-assistant/hassio) to see if someone is experian's something similar.

If you can't resolve it let me know, then we must find a way so I can get a copy of your image or access your PI an take a look.

**Comment by IKNout** [comment]
Eventually got round to trying this and '@marduv curl' worked for me, thanks for your help

**Comment by dale3h** [comment]
Closed via #10

**Comment by dale3h** [comment]
@Jafterdark Being that this script is supposed to be run from a fresh Raspbian image, you should be able to access the `configuration.yaml` directly via the Raspbian SSH (port 22 until further notice). After a successful execution of this script, the file should be located at `/usr/share/hassio/homeassistant/configuration.yaml`

1. Connect to the SSH server running on the base filesystem:
    ```bash
    $ ssh -p 22 pi@192.168.1.100
    ```
1. Open the `configuration.yaml` file for editing:
    ```bash
    $ sudo nano /usr/share/hassio/homeassistant/configuration.yaml
    ```
1. Add `hassio:` to the `configuration.yaml` file.
1. Save your changes by pressing `Ctrl+X`, `Y`, and then `Enter`
1. Restart your Raspberry Pi:
    ```bash
    $ sudo reboot
    ````
